
This fixes a logic bug to make the cite behaviour configurable.

  - Rene Rebe <rene@rocklinux.org>

--- mew-3.3/mew-draft.el	2003-06-19 05:58:13.000000000 +0000
+++ mew-3.3-fixed/mew-draft.el	2003-09-11 17:49:10.000000000 +0000
@@ -499,8 +499,8 @@
 	  (let ((mark-active t) (transient-mark-mode nil))
 	    (cond
 	     ;; arg will be effect in mew-cite-original
-	     ((and (mew-mark) (or (not mew-cite-ignore-mouse-region)
-				  (not (mew-mouse-region-p))))
+	     ((and (mew-mark) (and (not mew-cite-ignore-mouse-region)
+				   (not (mew-mouse-region-p))))
 	      (setq beg (region-beginning) end (region-end)))
 	     ((mew-header-p)
 	      ;; header exists in Message mode. Skip the header
